### ISSUE TITLE
Event indicator in SublimeDatePicker

### ISSUE THREAD

**Issue by WasimMemon** [vikramkakkar/SublimePicker#58]
Hi,
Thanks for the awesome library. This is really good library with all customization options.
My problem is for creating event i am using this date and time picker but i can't display events in date picker. for that i have to use some other library. So to reduce dependency and avoid 65k method problem i am trying to add event indicator in SublimeDatePicker. (display color single dot for events on particular date)

So is there any way i can do ?

**Comment by vvbhandare** [comment]
Did you get any solution?

**Comment by WasimMemon** [comment]
@vvbhandare No. For that I have used different library for display static calendar view with indicator.
